alembic/versions/94d288b1478e_initial_migration.py

```diff
@@ -0,0 +1,54 @@
+"""Initial migration
+
+Revision ID: 94d288b1478e
+Revises: 
+Create Date: 2024-11-04 08:39:02.698247
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = '94d288b1478e'
+down_revision: Union[str, None] = None
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table('cities',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('name', sa.String(length=255), nullable=True),
+    sa.Column('additional_info', sa.String(length=255), nullable=True),
+    sa.PrimaryKeyConstraint('id')
+    )
+    op.create_index(op.f('ix_cities_id'), 'cities', ['id'], unique=False)
+    op.create_index(op.f('ix_cities_name'), 'cities', ['name'], unique=True)
+    op.create_table('temperatures',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('city_id', sa.Integer(), nullable=True),
+    sa.Column('date_time', sa.DateTime(), nullable=True),
```

Comment:
Consider whether `city_id` and `date_time` should be nullable. If these fields are essential for your data model, you might want to set `nullable=False` to ensure data integrity.